distribution/docs/src/main/resources/_contents/_security-framework/security-idp-intro-contents.adoc

```diff
@@ -0,0 +1,28 @@
+
+=== Security IdP
+
+The Security IdP application provides service provider handling that satisfies the http://docs.oasis-open.org/security/saml/v2.0/saml-profiles-2.0-os.pdf[SAML 2.0 Web SSO profile] in order to support external IdPs (Identity Providers).
```

Comment:
external IdPs or SPs (service providers). We should say that this capability allows someone to use DDF as the SSO solution for their entire enterprise.